File: Introduction/C implementation/uniqueElements.c
//Program to check if all the elements are distinct in a array
#include <stdio.h>

int elementsDistinct(int A[], int size);
int main() 
{
    int array[] = {1, 2, 3, 4, 5};
    int size = sizeof(array) / sizeof(array[0]);
    if (elementsDistinct(array, size)) 
    {
        printf("Elements in the array are distinct\n");
    } 
    else 
    {
        printf("Elements in the array are not distinct\n");
    }
    return 0;
}

int elementsDistinct(int A[], int size) 
{
    for (int i = 0; i < size - 1; i++) 
    {
        for (int j = i + 1; j < size; j++) 
        {
            if (A[i] == A[j])
            {
                return 0;
            }
        }
    }
    return 1;
}

File: Unit - 1/Introduction/Java Implementation/UniqueElements.java
//Program to check if all the elements are distinct in a array
public class UniqueElements {

    private static boolean elementsDistinct(int[] A) {
        for (int i = 0; i < A.length - 1; i++) {
            for (int j = i + 1; j < A.length; j++) {
                if (A[i] == A[j]) {
                    return false;
                }
            }
        }
        return true;
    }

    public static void main(String[] args) {
        int[] array = {1, 2, 3, 4, 1, 5};
        if(elementsDistinct(array) == true){
            System.out.println("Elements in the array are distinct");
        } else{
            System.out.println("Elements in the array are not distinct");
        }
    }
}